frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -543,7 +543,7 @@
   "An error occurred Pool {name} was not created": "An error occurred Pool {name} was not created",
   "Pool {name} creation timed out. Please check if ocs-operator and rook operator are running": "Pool {name} creation timed out. Please check if ocs-operator and rook operator are running",
   "The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.": "The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.",
```

Comment:
The creation of an OpenShift Container Storage cluster is still in progress or has failed. Try again after the storage cluster is ready to use.